run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+cd redcap_docker
+docker compose up -d
+
+cd ..
+
+cd redcap_cypress
+npm install
+rm -r cypress/fixtures/*
```

Comment:
I think we need to view established subfolders (cdisc_files, dictionaries, import_files) of `/Files/` folder as **reserved for only redcap_rsvc features**.

I'd therefore recommend 
```
rm -r cypress/fixtures/cdisc_files cypress/fixtures/dictionaries cypress/fixtures/import_files
``` 

instead of deleting everything in that folder.

The problem with deleting everything is that users who are creating their own institution-specific features will inevitably have files they need to store in fixtures and this command would wipe out those files.

Reply:
@aldefouw, that makes perfect sense.  I just updated this command, and added "-f" to prevent warning messages on the first run (when those folders don't exit).